internal/config/config.go

```diff
@@ -100,24 +100,37 @@ func NewConfig(app string) (*Config, error) {
 	if err != nil {
 		return nil, err
 	}
-	err = c.configDatabase()
-	if err != nil {
-		return nil, err
-	}
-
 	switch app {
 	case "ingest":
 		c.configInbox()
 		c.configArchive()
 		c.configCrypt4gh()
+		err = c.configDatabase()
+		if err != nil {
```

Comment:
Not really to be addressed here, but we might want to consider if configDatabase should just give up and fatal if it can't get things up).

Reply:
These changes where introduced by Dmytro inorder to propagate the error back to the main function.

Reply:
Yep. I'm a bit hesitant about panicking, because it's usually hard to test. I prefer propagating errors upwards where possible.

Reply:
I kind of agree about the issues, but don't like the disharmony. But absolutely, this is fine for now.